Bona Nova: Version 4.001; ttfautohint (v1.8.3) added

RosaWagner opened the pull request:
54711ed: [gftools-packager] Bona Nova: Version 4.001; ttfautohint (v1.8.3) added

* Bona Nova Version 4.001; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/kosmynkab/Bona-Nova at commit https://github.com/kosmynkab/Bona-Nova/commit/fdc7db4c1fcd4d63a922ea8797881a39d3a62caa.

 7cbdb00: [gftools-packager] ofl/bonanova remove METADATA "source".  google/fonts#2587

RosaWagner commented:
I don't understand, either we have two different reports, or we have two different fonts… I'll investigate.

RosaWagner commented:
Updated

## Bona Nova: Version 4.001; ttfautohint (v1.8.3) added

---
 cbf01e5: [gftools-packager] Bona Nova: Version 4.001; ttfautohint (v1.8.3) added

* Bona Nova Version 4.001; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/kosmynkab/Bona-Nova at commit https://github.com/kosmynkab/Bona-Nova/commit/f7d909d11c882f4d83a2c53e1d2df1733308812f.

 53e817d: [gftools-packager] ofl/bonanova remove METADATA "source".  google/fonts#2587

RosaWagner commented on ofl/bonanova/METADATA.pb:
```diff
@@ -0,0 +1,40 @@
+}
+fonts {
+  name: "Bona Nova"
+  style: "normal"
```
Why ? Unless I forgot something, the font has all the necessary bits and style conventions… is this a packager issue?

m4rc1e commented:
FontBakery is complaining because the macStyle bit hasn't been set.

m4rc1e commented:
Sorry, I've just worked out why this is failing.

The designer's url is `https://capitalics.wtf`. Since it contains `italic`, the check `com.google.fonts/check/metadata/canonical_style_names` thinks this is an Italic font. The fonts are fine, the check is wrong! Let's merge and I'll open an issue in Fontbakery.

RosaWagner commented:
Updated

## Bona Nova: Version 4.001; ttfautohint (v1.8.3) added

---
 1b294ab: [gftools-packager] Bona Nova: Version 4.001; ttfautohint (v1.8.3) added

* Bona Nova Version 4.001; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/kosmynkab/Bona-Nova at commit https://github.com/kosmynkab/Bona-Nova/commit/a5cbdfb8741af20ea5bfe252f0128beed6c8beed.

 86ad7ea: [gftools-packager] ofl/bonanova remove METADATA "source".  google/fonts#2587
